lib/LANraragi/Plugin/DateAdded.pm

```diff
@@ -0,0 +1,53 @@
+package LANraragi::Plugin::DateAdded;
+
+use strict;
+use warnings;
+
+#Plugins can freely use all Perl packages already installed on the system 
+#Try however to restrain yourself to the ones already installed for LRR (see tools/cpanfile) to avoid extra installations by the end-user.
+use Mojo::UserAgent;
+
+use LANraragi::Model::Plugins;
+
+#Meta-information about your plugin.
+sub plugin_info {
+
+    return (
+        #Standard metadata
+        name  => "Date Added",
+        namespace => "DateAddedPlugin",
+        author => "Utazukin",
+        version  => "0.1",
+        description => "Adds the date the archive was added as a tag.",
+        oneshot_arg => "Use file modified time.",
```

Comment:
I'd like a better description for end-users -- Stuff like under which namespace the date ends up, and how should the argument be written for it to be picked up by the plugin ("yes"? "true"? It's both here if I read correctly)

I admit the current system for boolean arguments sucks, but I'm not sure how to make it work best with the current format. Suggestions welcome!